modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
+number of CPUs within the host.
+
+Server resources deployed at the edge, such as cores, are expensive and limited. Customers require nearly all cores for their application workloads. The resources consumed by infrastructure is a key portion of the customer's business
+case for the selection of a vRAN infrastructure. A hypothetical DU example is an unusually resource-intensive workload, typically requiring 20 dedicated cores.
+
+This feature implements the necessary OCP features and cluster configuration to isolate all housekeeping tasks and infrastructure pods to a partitioned set of CPU resources on a Single Node OpenShift (SNO) cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All housekeeping tasks and infrastructure pods operate solely on that cpuset.
+
+* Tunes the cluster configuration (with the applied DU profile) so the actual CPU usage fits within the assigned cpuset.
+
+[NOTE]
+====
+This feature is only available on SNO in this release.
+====
+
+The minimum number of reserved CPUs required for the management partition for an SNO is four CPUs/HTs. Inclusion of operators or workloads beyond the set specified in “Management pods” requires additional CPU/HTs.
+
+Workload partitioning isolates the workloads away from the non-management workloads using the normal scheduling capabilities of kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing management and normal workloads.
+
+To fully leverage this feature, you need to deploy the Performance Addon Operator and apply the performance profile as part of day 2 configuration.
```

Comment:
suggestion: replace "this feature" with "workload partitioning"

I don't believe "day 2" is a customer-facing term.  Can you sidestep that with something like "...deploy the Performance Addon Operator and apply the performance profile after completing the installation of the cluster."  Or, deploying the Operator might already imply that the cluster is already installed and it's an operational configuration timeframe.